fdk/event_handler.py

```diff
@@ -50,3 +54,16 @@ def event_handle(handle_code):
         )
 
     return pure_handler
+
+
+def log_frame_header(headers):
+    framer = os.environ.get(constants.FN_LOGFRAME_NAME)
+    if framer is None:
+        return
+    value_src = os.environ.get(constants.FN_LOGFRAME_HDR)
+    if value_src is None:
+        return
+    id = headers.get(value_src.lower())
+    if id is not None:
```

Comment:
```python
framer = os.environ.get(constants.FN_LOGFRAME_NAME)
if frame is not None:
    value_src = os.environ.get(constants.FN_LOGFRAME_HDR)
    if value_src is not None:
        if value_src.lower() in headers:
            id = headers.get(value_src.lower())
            frm = "\n{}={}\n".format(framer, id)
            print(frm, file=sys.stderr, flush=True) # <--- this will guarantee that whatever goes to file will flushed right away
            print(frm, file=sys.stdout, flush=True)

```